src/tests/panelSet.test.ts

```diff
@@ -0,0 +1,62 @@
+import { resolvePlugin } from '@babel/core';
+import { _createPanelSetController, _getPanelSetByIDController, _getAllPanelSetsFromUserController } from '../requestHandlers/panelSet';
+
+import * as panelSetService from '../services/panelSetService';
+import * as userService from '../services/userService';
+
+import { Sequelize } from 'sequelize';
+jest.mock('../services/panelSetService');
+jest.mock('../services/userService');
+
+
+let sequelizeMock: jest.Mocked<Sequelize>;
+beforeEach(() => {
+    sequelizeMock = {} as jest.Mocked<Sequelize>;
+});
+const panelData = {id: 'panelsetID'}
+
+describe("_createPanelSetController", () => {
```

Comment:
Since we are testing specifically the controller, I think we should keep the word `controller` somewhere in the name. Just in case we want to test something else regarding a command. I will use your current naming convention and append the word `controller`